pkg/vm/tests/vm_test.go

```diff
@@ -70,6 +68,23 @@ func newStoragePlugin() *storagePlugin {
 	}
 }
 
+func (s *storagePlugin) getInterop(id uint32) *vm.InteropFuncPrice {
+	var f vm.InteropFunc
+	if id == vm.InteropNameToID([]byte("Neo.Storage.Get")) {
```

Comment:
Wouldn't it be more readable (and have less unnecessary conversions) with `switch`?

Reply:
Sure.